core/consensus/aura/src/lib.rs

```diff
@@ -447,13 +450,14 @@ fn check_header<C, B: BlockT, P: Pair>(
 }
 
 /// A verifier for Aura blocks.
-pub struct AuraVerifier<C, P> {
+pub struct AuraVerifier<C, P, T> {
 	client: Arc<C>,
 	phantom: PhantomData<P>,
 	inherent_data_providers: inherents::InherentDataProviders,
+	transaction_pool: Option<Arc<T>>,
```

Comment:
This should probably be replaced with a `F: Fn(Block::Extrinsic)` which will just push the extrinsic to the transaction pool As it stands you can't do anything with this `T`, and it's probably easier to not drag any complicated `TransactionPool` types here.

Reply:
I'l add add a trait that implements `submit_one` to the transaction pool, as per offline discussion :).